### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by kvasukib**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+A declarative library for application development using cloud services.
+
+## 0.9.0
+
+Initial release! Includes Core features, plus support for these categories:
+
+- Analytics
+- API
+- DataStore
+- Hub
```
Comment (royjit) [inline CHANGELOG.md]: Hub is implemented? I saw few places this todo like this `//TODO: Dispatch to hub`

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- Logging
```
Comment (raphkim) [inline CHANGELOG.md]: Are these currently supported?

File: CHANGELOG.md (hunk 1 above)
Comment (kvasukib) [reply CHANGELOG.md]: Hub as an independent category is supported and implemented. However Hub is not used by other categories.

File: CHANGELOG.md (hunk 2 above)
Comment (kvasukib) [reply CHANGELOG.md]: We have plugins for those categories which customers can use.

**Pull request merged**